gammapy/cube/psf_map.py

```diff
@@ -315,17 +315,17 @@ def containment_radius_map(self, energy, fraction=0.68):
 
         return m
 
-    def stack(self, other):
+    def stack(self, other, in_place=True):
```

Comment:
I think the `in_place` option is not used at the moment. Another suggestion: maybe change to a `copy` option instead? So Always return the stacked map, but in one case a copy is made, whereas in the other `self` is returned. I think this corresponds to the API we have in other places as well e.g. `Map.cutout()`.

Reply:
Done.